### PR TITLE
www-apps/radicale: bump to 3.2.3 and BDEPEND fix for test

### PR DESCRIPTION
**Pull request by henning-schild**
---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: www-apps/radicale/radicale-3.2.2.ebuild [hunk 1]
```diff
@@ -35,7 +35,7 @@ RDEPEND="
 
 BDEPEND="
 	test? (
-		<dev-python/pytest-8[${PYTHON_USEDEP}]
+		dev-python/pytest[${PYTHON_USEDEP}]
```
Comment (henning-schild) [inline www-apps/radicale/radicale-3.2.2.ebuild:38]: Let me know if this requires a new revision. I think it does not, but am not sure.

Comment (mgorny) [reply www-apps/radicale/radicale-3.2.2.ebuild:38]: It's fine in place.

**Comment by mgorny** [comment]
> adjust BEDEPEND for test target

To be or not to be a BDEPEND xP (I'll fix the typo).

**Comment by mgorny** [comment]
Ok, tests passed. Thanks!

**Comment by henning-schild** [comment]
> Ok, tests passed. Thanks!

Thank you!

**Pull request closed without merging**